### ISSUE TITLE
Adding fields to a new form fails

### ISSUE THREAD

**Issue by boboldehampsink** [barrelstrength/craft-sprout-forms#255]
### Description

When dragging a field to a new form I get the following error

```json
{"error":"SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"sproutformscontent_form1\" does not exist\nThe SQL being executed was: ALTER TABLE \"sproutformscontent_form1\" ADD \"field_email1\" varchar(255)"}
```

### Steps to reproduce

1. Create a new form
2. Drag a field to it
3. Inspect the occurring ajax request

### Additional info

- Craft version: 3.1.18
- PHP version: 7.3.3
- Database driver & version: PostgreSQL 10.7
- Plugins & versions: Sprout Forms 3.0.0-beta.46

**Comment by BenParizek** [comment]
Thanks for reporting @boboldehampsink. We'll try to recreate and let you know.

**Comment by boboldehampsink** [comment]
Added a fix in PR #260

**Comment by BenParizek** [comment]
Thanks @boboldehampsink. Just pushed a new release with this resolved: v3.0.0-beta.49
